### PR TITLE
credits prototype

### PR DESCRIPTION
**Pull request by xleonx0x**
click le button to show le credits

### CODE REVIEW

File: frontend/src/components/Credits/Credits.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+function Credits() {
+  return (
+    <div className={classes.container}>
+      <div className={classes.box}>
+        <div className={classes.title}>Credits</div>
+        Made with the most amazing team:
+        <div className={classes.creditstext}>
+          <ul>
+            <li>Lachlan Shoesmith: Master of Yapping</li>
```
Comment (lachlanshoesmith) [inline frontend/src/components/Credits/Credits.tsx]: lol

File: frontend/src/components/Credits/Credits.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            <li>Ben Goldwin: </li>
+            <li>Chris Wong: </li>
+            <li>Alyssa Cheong: </li>
+            <li>Oswaldo Prajitno: </li>
```
Comment (lachlanshoesmith) [inline frontend/src/components/Credits/Credits.tsx]: osvaldo

File: frontend/src/components/Credits/Credits.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+function Credits() {
+  return (
+    <div className={classes.container}>
+      <div className={classes.box}>
+        <div className={classes.title}>Credits</div>
+        Made with the most amazing team:
+        <div className={classes.creditstext}>
+          <ul>
+            <li>Lachlan Shoesmith: Master of Yapping</li>
+            <li>Ben Goldwin: </li>
```
Comment (lachlanshoesmith) [inline frontend/src/components/Credits/Credits.tsx]: godwin

File: frontend/src/components/Credits/Credits.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import classes from "./Credits.module.scss";
+
+function Credits() {
+  return (
+    <div className={classes.container}>
+      <div className={classes.box}>
+        <div className={classes.title}>Credits</div>
```
Comment (lachlanshoesmith) [inline frontend/src/components/Credits/Credits.tsx]: make it a h1 dood

File: frontend/src/components/Navbar/Navbar.tsx [hunk 5]
```diff
@@ -1,21 +1,37 @@
...
         <button className={classes.hover}>Help</button>
       </div>
       <div>
-        <button className={classes.hover}>Credits</button>
+        <button className={classes.hover} onClick={toggleCredits}>
+          Credits
+        </button>
         <ProfileIcon url="/yellowshirt.svg" />
       </div>
+      {showCredits &&
```
Comment (lachlanshoesmith) [inline frontend/src/components/Navbar/Navbar.tsx:30]: i do worry this approach is a bit less readable than just using a native <dialog> but its all good

**Review by lachlanshoesmith — CHANGES_REQUESTED** [review]

File: frontend/src/components/Credits/Credits.tsx (hunk 4 above)
Comment (xleonx0x) [reply frontend/src/components/Credits/Credits.tsx]: ok dood

**Review by lachlanshoesmith — APPROVED** [review]

**Review by lachlanshoesmith — APPROVED** [review]

**Review by lachlanshoesmith — APPROVED** [review]

**Pull request merged**